api/lib/infrastructure/repositories/cpf-certification-result-repository.js

```diff
@@ -45,6 +45,7 @@ function _selectCpfCertificationResults() {
     )
     .where('certification-courses.isPublished', true)
     .where('certification-courses.isCancelled', false)
+    .whereNotNull('certification-courses.sex')
```

Comment:
**suggestion :** En local tous les tests passent si je vire ces 3 lignes

Reply:
Il y a un soucis avec tout les tests **should return an empty array**, il est important de s'assurer qu'un test echoue sans les condition necessaire pour le cas precis du test avant de le faire passer.
A corriger en mob

Reply:
On a corrige (c'est fixupe, trop tard, mais ca aurait pu etre interessant d'etre sur un commit a part pour mettre en valeur le probleme et ca correction)